### DISCUSSION TITLE
Serene template in VS Gallery is not updated frequently compared to Serenity Repository

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#6631] [General]
Please consider updating Serene template if an update takes place in Serenity repository's CHANGELOG.  Otherwise, it seems that the template is out of date.

Thank you

**Comment by VictorTomaili** [comment]
Most of the updates affect the NuGet packages, to update NuGet packages, we are not publishing a new version of template. You can update NuGet packages yourself.
